packages/contracts-bedrock/test/setup/CommonTest.sol

```diff
@@ -76,6 +76,12 @@ contract CommonTest is Test, Setup, Events {
         // Deploy L2
         Setup.L2();
 
+        // Authorize portals to interact with the SharedLockbox.
+        vm.startPrank(address(superchainConfig));
+        sharedLockbox.authorizePortal(address(optimismPortal));
```

Comment:
Do we need this line? Not sure since the lockbox integration doesn't exist on the older portal implementation

Reply:
It is being used on some tests so we need this. But i think in the `develop` branch the portal is going to be deprecated, so we can remove it later.

Reply:
Perfect then